.circleci/config.yml

```diff
@@ -95,7 +95,7 @@ commands:
 jobs:
   install-operator:
     docker:
-      - image: cimg/go:1.20
+      - image: cimg/go:1.22
```

Comment:
please rebase and switch images to: quay.io/3scale/operators-ci-images:v1.22

Reply:
Done. Thank you